src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -9,7 +9,8 @@
         <label><strong>Name Translation(s)</strong></label>
         <v-flex md1>
             <v-chip v-if="hasNameTranslationChange" x-small label color="#1669BB" text-color="white">
-                Corrected
+              <span v-if="isCorrectionView">Corrected</span>
+              <span v-else>Changed</span>
```

Comment:
Use `editedLabel()` in common mixin instead.

Reply:
Another option is to replace this block with `<action-chip :props />`. 
It's a shared component now and is essentially this code. 
Super small, but it's there if you like :)

Reply:
Thanks, great, I did use this.